design/proposals/load_policy/load_policy_validation.md

```diff
@@ -0,0 +1,257 @@
+# Solution Design - Load policy validation
+
+## Table of Contents
+* [Glossary](#glossary)
+* [Useful Links](#useful-links)
+* [Issue Description](#issue-description)
+* [Solution](#solution)
+   + [Solution - scope](#solution---scope)
+   + [User Interface](#user-interface)
+* [Design](#design)
+   + [Flow Diagrams](#flow-diagrams)
+   + [Sequence Diagrams](#sequence-diagrams)
+      - [Sequence Diagrams - additional info](#sequence-diagrams---additional-info)
+* [Performance](#performance)
+* [Backwards Compatibility](#backwards-compatibility)
+* [Affected Components](#affected-components)
+* [Test Plan](#test-plan)
+   + [Out of Scope](#out-of-scope)
+   + [Prerequisites](#prerequisites)
+* [Documentation](#documentation)
+* [Security](#security)
+* [Infrastructure](#infrastructure)
+* [Audit](#audit)
+* [Open Questions](#open-questions)
+* [Delivery Plan](#delivery-plan)
+   + [Total 1st milestone EE 45](#total-1st-milestone-ee-45)
+      - [Ramp-up EE 2](#ramp-up-ee-2)
+      - [Designs EE 2](#designs-ee-2)
+      - [Implementations EE 19](#implementations-ee-19)
+      - [Testing EE 13](#testing-ee-13)
+      - [Security EE 5](#security-ee-5)
+      - [Docs EE 3](#docs-ee-3)
+      - [Demo EE 1](#demo-ee-1)
+* [Definition of Done](#definition-of-done)
+* [Solution Review](#solution-review)
+
+## Useful Links
+- [Feature doc](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-8566)
+- [UX research page](https://ca-il-confluence.il.cyber-ark.com.mcas.ms/display/rndp/JWT+Authenticator+-+load+policy+validation)
+- [JWT authenticator stage 1.1 feature doc](https://ca-il-confluence.il.cyber-ark.com.mcas.ms/display/rndp/JWT+Authenticator+-+re-scope#JWTAuthenticatorrescope-Stage1.1-SimplegenericJWTauthenticatorwithtokenschema)
+
+
+## Issue Description
+In misconfiguration errors, for example:
+
+for hosts without JWT **mandatory** annotation,
```

Comment:
why only without mandatory fields?
We should validate all cases

Reply:
sure this is just an example,
the use cases validation are not changing the high level design